server/models/MessageModel.js

```diff
@@ -33,6 +33,9 @@ const MessageSchema = new Schema(
       type: Boolean,
       default: false,
     },
+    replyTo: {
+      type: Schema.Types.ObjectId
+    }
```

Comment:
why not store the message that it's replying to

Reply:
Storing the message means it won't be dynamic any more.

Reply:
How so?

Reply:
either

Reply:
The latter is the reason for my previous explanation, the former is good, but doesn't really offer any benefits since the field is only needed at the frontend.

Reply:
i rather it's added, because the messageModel will be used for the friends feature.